src/test/java/com/amazon/opendistroforelasticsearch/search/asynchronous/utils/RestTestUtils.java

```diff
@@ -71,9 +72,11 @@
     public static Request buildHttpRequest(SubmitAsynchronousSearchRequest submitAsynchronousSearchRequest) throws IOException {
 
         SearchRequest searchRequest = submitAsynchronousSearchRequest.getSearchRequest();
+        String baseUri = Randomness.get().nextBoolean() ? AsynchronousSearchPlugin.BASE_URI
```

Comment:
Do we just need to change it here?

Reply:
added this change in delete and get apis. thanks for catching that!